examples/open-source-101/test/development.spec.ts

```diff
@@ -0,0 +1,65 @@
+import supertest from 'supertest';
+import { WireMock } from 'wiremock-captain';
+import spotifyGetArtistResponse from './../src/spotify-get-artist-200-resp.json';
+
+const app = supertest('http://localhost:8080');
+
+describe('/artist-popularity', () => {
+    const mockServer = new WireMock('http://localhost:8085');
+
+    afterEach(() => {
+        // clear mocks after each test
+        return mockServer.clearAllExceptDefault();
+    });
+
+    test('should respond back with successful response when spotify API is working', async () => {
+        // register successful Spotify API response
+        await mockServer.register(
+            // incoming request schema
+            {
+                method: 'GET',
+                endpoint: '/v1/artists/0TnOYISbd1XYRBk9myaseg',
+            },
+            // error response schema
```

Comment:
this is the happy path right? Not error response schema

Reply:
whooops